src/components/crear_actividad/createActivity.js

```diff
@@ -10,254 +10,304 @@ import Link from '@material-ui/core/Link';
 import Typography from '@material-ui/core/Typography';
 import { ChooseActivity } from './chooseActivity';
 import { Participants } from './participants';
-import {Verify} from './verify';
+import { Verify } from './verify';
 import Menu from "../menu/NavBar";
 import { withStyles } from '@material-ui/core/styles';
+import axios from "axios"
+import axiosHeader from '../../api/axiosHeader';
 
 
+class CreateActivity extends React.Component {
 
-class CreateActivity extends React.Component{
-
-    constructor(props){
-        super(props)
-        this.state = {activity: "", date: moment(),time:Date.now(), location:"",bet:"",stateBet: false,checkParticipants:false, checkTeams: false , checked: [],count:0, checkIndividual: false};
-        //metodos para chooseActivity 
-        this.handleChangeActivity = this.handleChangeActivity.bind(this);
-        this.handleDateChange = this.handleDateChange.bind(this);
-        this.handleTimeChange = this.handleTimeChange.bind(this);
-        this.handleChangeLocation = this.handleChangeLocation.bind(this);
-        this.handleChangeBet = this.handleChangeBet.bind(this);
-        this.handleChangeStateBet = this.handleChangeStateBet.bind(this);
-        //metodos para participants
-        this.handlecheckParticipants = this.handlecheckParticipants.bind(this);
-        this.handlecheckTeams = this.handlecheckTeams.bind(this);
-        this.handleToggle = this.handleToggle.bind(this);
-        this.handlecheckIndividual = this.handlecheckIndividual.bind(this);
-        //metodos para createActivity 
-        this.handleNext = this.handleNext.bind(this);
-        this.handleBack = this.handleBack.bind(this);
-        this.nextStep = this.nextStep.bind(this);
-    }
-  
-    
-    render(){
-       function Copyright() {
-            return (
-              <Typography variant="body2" color="textSecondary" align="center">
-                {'Copyright © '}
-                <Link color="inherit" href="https://material-ui.com/">
-                  Your Website
+  constructor(props) {
+    super(props)
+    this.state = { activity: "", date: new Date(), time: new Date(), location: "", bet: "", description: "", stateBet: false, checkParticipants: false, checkTeams: false, checked: [], count: 0, checkIndividual: false };
+    //metodos para chooseActivity 
+    this.handleChangeActivity = this.handleChangeActivity.bind(this);
```

Comment:
too many handlers in one component may indicate your are trying to support more than one responsibility at a time. Try to keep your components small and simple